website/docs/guides/migration/versions/04-upgrading-to-v1.4.md

```diff
@@ -27,7 +27,16 @@ Relevant changes:
 
 ### For maintainers of adapter plugins
 
-Link to GH discussion forthcoming
+> **TL;DR** Not much heavy lifting for this minor version. But FYI there will be more work for `1.5.0`. Be warned!
```

Comment:
Should we take a more proactive tone here? v1.5 will require more work, and here's how we plan to help. Just an idea:
```suggestion
> **TL;DR** Not much heavy lifting for this minor version. We anticipate more work for `1.5.0`. We plan to release betas early & often, and provide guidance on upgrading.
```